cmd/soroban-rpc/internal/ledgerentry_storage/migrations/01_init.sql

```diff
@@ -10,11 +10,12 @@ CREATE TABLE metadata (
     value TEXT NOT NULL
 );
 
-
-
-CREATE INDEX ledger_entries_key ON ledger_entries(key);
-
+CREATE TABLE ledger_close_meta (
```

Comment:
I don't see why both the ledger entries and the events needs to live in the same database; they have quite a distinct an unrelated "characteristics" and not ever accessed together.
I'm concerned that placing both in the same database would result in a database-level locking between the two that isn't needed.

Reply:
I do like the fact that we can commit the entire ledger changes in a single commit. In this particular case, I don't believe that it really give us much advantage. The events are only being "added" and not really requiring any synchronization with the rest of the data. You have already noticed that the current ledger data tend to become very large. That's because we have lots of write/delete. In order to avoid that, we need to perform periodic full vacuuming.
Unfortunately, performing the full vacuuming on a large database would take a long time - which is why it's advisable to break out the tables that have a different insert/delete profile into a separate database files.

Reply:
That sounds like the right thing to do. It would allow us to release this as a working prototype while we improve on the implementation.